README.md

```diff
@@ -95,298 +56,26 @@ To run a Watcher only, first make sure you sent an ENV variable called with `INF
 docker-compose -f docker-compose-watcher.yml up
 ```
 
-### Get the deployed contract details
-
-`curl localhost:5000/get_contract`
-
 ### Troubleshooting Docker
 You can view the running containers via `docker ps`
 
 If service start up is unsuccessful, containers can be left hanging which impacts the start of services on the future attempts of `docker-compose up`.
 You can stop all running containers via `docker kill $(docker ps -q)`.
 
-If the blockchain services are not already present on the host, docker-compose will attempt to build the image with the tag `elixir-omg:dockercompose` and continue to use that.
-If you want Docker to use the latest commit from `elixir-omg` you can trigger a fresh build by passing the `--build` flag to `docker-compose up --build`.
+If the blockchain services are not already present on the host, docker-compose will attempt to pull the latest build coming from master.
+If you want Docker to use the latest commit from `elixir-omg` you can trigger a fresh build by building all three services with `make docker-childchain`, `make docker-watcher` and `make docker-watcher_info`.
 
-## Install on a Linux host & manual start up
-Follow the guide to **[install](docs/install.md)** the child chain server and watcher.
-Then use the guide in **[manual service startup](docs/manual_service_startup.md)** to stand up.
-
-### Follow the demos
-After starting the child chain server and/or Watcher as above, you may follow the steps in the demo scripts.
-Note that some steps should be performed in the Elixir shell (iex) and some in the shell directly.
-
-To start a configured instance of the `iex` REPL, from the `elixir-omg` root directory inside the container do:
-```bash
-iex -S mix run --no-start --config ~/config.exs
-```
-
-Follow one of the scripts in the [docs](docs/) directory. Don't pick any `OBSOLETE` demos.
+# Install on a Linux host
+Follow the guide to **[install](docs/install.md)** the Child Chain server, Watcher and Watcher Info.
 
 ## Troubleshooting
 Solutions to common problems may be found in the [troubleshooting](docs/troubleshooting.md) document.
```

Comment:
Not caused by this PR but can you also help fix this broken link? 🙏

Reply:
hm... interesting. we don't have any docs for that :D

Reply:
I'll just delete this!